timewarp_gl/plugin.cpp

```diff
@@ -235,7 +427,7 @@ class timewarp_gl : public threadloop {
                     // Set the physical distortion mesh coordinates. These are rectangular/gridlike, not distorted.
                     // The distortion is handled by the UVs, not the actual mesh coordinates!
                     distortion_positions[eye * num_distortion_vertices + index].x =
-                        (-1.0f + eye + ((float) x / hmdInfo.eyeTilesWide));
+                        (-1.0f + 2.0f * ((float) x / hmdInfo.eyeTilesWide));
```

Comment:
What's the reason for this change?

Reply:
The difference here depends on how we construct the timewarp mesh. From what I remember, the previous timewarp assumed that it was operating on a single buffer, so it would need two distinct meshes. In normalized device coordinates, the x-axis of the left eye would range from [-1.0, 0] while the x-axis of the right eye would range from [0, 1.0]. 

On the other hand, when we run timewarp on each eye, we want to construct the mesh to occupy the entire width, i.e. an NDC range of [-1.0, 1.0].